frontends/web/src/components/guide/guide.tsx

```diff
@@ -107,6 +74,6 @@ function Guide({ screen, shown, t, children }: RenderableProps<Props>): JSX.Elem
     );
 }
 
-const HOC = translate<ProvidedProps>()(share<SharedProps, ProvidedProps & TranslateProp>(store)(Guide));
+const HOC = translate<{}>()(share<SharedProps, TranslateProp>(store)(Guide));
```

Comment:
`translate` has a default generic type of `{}` and thus you should be able to skip all the generics of `translate`: https://github.com/digitalbitbox/bitbox-wallet-app/blob/e144c48cdee32739fd144ba4a2488d190c6fd083/frontends/web/src/decorators/translate.ts#L66